data/_settings/datacatalogue/molgenis_settings.csv

```diff
@@ -0,0 +1,3 @@
+key,value
+menu,"[{""label"":""Catalogue"",""href"":""catalogue"",""key"":""nvleta"",""submenu"":[],""role"":""Viewer""},{""label"":""SSR Catalogue"",""href"":""ssr-catalogue"",""key"":""blabla"",""submenu"":[],""role"":""Viewer""},{""label"":""Tables"",""href"":""tables"",""role"":""Viewer"",""key"":""zushli"",""submenu"":[]},{""label"":""Schema"",""href"":""schema"",""role"":""Manager"",""key"":""ndi9r"",""submenu"":[]},{""label"":""Up/Download"",""href"":""updownload"",""role"":""Editor"",""key"":""f427rf"",""submenu"":[]},{""label"":""Graphql"",""href"":""graphql-playground"",""role"":""Viewer"",""key"":""rdhauq"",""submenu"":[]},{""label"":""Settings"",""href"":""settings"",""role"":""Manager"",""key"":""9le9g9"",""submenu"":[]},{""label"":""Help"",""href"":""docs"",""role"":""Viewer"",""key"":""i11sn"",""submenu"":[]}]"
```

Comment:
we probably should rethink how settings are serialized because all this escaping sucks ;-)

Reply:
indeed a 'nice to have'